.github/workflows/config/mlc_config.json

```diff
@@ -2,35 +2,53 @@
     "ignorePatterns": [
         {
             "pattern": "https://github.com/Your_Github_Handle/owasp-mstg"
-        },{
+        },
+        {
             "pattern": "changelog"
-        },{
+        },
+        {
             "pattern": "https://github.com/commjoen/contributors-mstg"
-        },{
+        },
+        {
             "pattern": "http://127.0.0.1:8000"
-        },{
+        },
+        {
             "pattern": "https://haveibeenpwned.com"
-        },{
+        },
+        {
             "pattern": "https://build.frida.re/"
-        },{
+        },
+        {
             "pattern": "http://apt.thebigboss.org/repofiles/cydia/"
-        },{
+        },
+        {
             "pattern": "https://build.frida.re"
-        },{
+        },
+        {
             "pattern": "http://cydia.radare.org/debs/"
-        },{
+        },
+        {
             "pattern": "https://cydia.akemi.ai/"
-        },{
+        },
+        {
             "pattern": "https://cydia.angelxwind.net"
+        },
+        {
+            "pattern": "https://developers.android.com"
+        },
+        {
+            "pattern": "https://source.android.com"
         }
-
     ],
-    "replacementPatterns": [
+    "httpHeaders": [
         {
-            "pattern": "^.attachments",
-            "replacement": "file://some/conventional/folder/.attachments"
+            "urls": [
+                "https://",
+                "http://"
+            ],
+            "headers": {
+                "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0"
+            }
         }
-    ],
-    "see":" https://github.com/tcort/markdown-link-check#config-file-format"
```

Comment:
Please keep this line which directly points to the official config. options.

Reply:
Sure. The lines that were in this files were the default ones, so I thought that maybe they were forgotten.